### ISSUE TITLE
Phone continuous data collection

### ISSUE THREAD

**Issue by nipunbatra** [nipunbatra/Home_Deployment#40]
Just checked on one of the phones, FunF had data close to 700 MB, while on another 30 MB. Both have been running since same date.
How do i verify? Do i need to export and check?

@shailja1275 : Please see

**Comment by shailja-thakur** [comment]
yes , possibly even I cant infer just from the size. After seeing only
can get to know the reason , whether the phone wan on /off  or whatever.

**Comment by nipunbatra** [comment]
Do you have some script to check the same? Since i don't think i will want to upload 30+ MB data.

**Comment by shailja-thakur** [comment]
I have python scripts , it hardly takes minute to upload

**Comment by nipunbatra** [comment]
Please put the scripts on github/ gists and share the link here

**Comment by shailja-thakur** [comment]
I am having it on dropbox , I will share in a moment.

**Comment by nipunbatra** [comment]
None of the export options are feasible at my net speed!
- Can i do a local transfer on LAN?
- You told it uses sqlite. Since sqlite is a file based database, if i just copy the file, it should work. Can you try this and let me know?

**Comment by shailja-thakur** [comment]
Yes , do try ,I never tried it though it should work , on phone there must
be db files and additionally the zipped file

**Comment by nipunbatra** [comment]
I meant can **you** @shailja1275  please try and let me know. Just tell me the location of sqlite and how to copy it to my laptop. Remaining things i will do.

**Comment by shailja-thakur** [comment]
ok , acually i uninstalled funf from my phone long back , i will do it all
again and tell you in 15 min

**Comment by shailja-thakur** [comment]
As far as I remember , in file manager there is a folder name starting with
mit.  (a very long name) within that there are .db files ,just download
them.

Actual path I will let you know in short while

**Comment by nipunbatra** [comment]
Still having issues with accessing data (propreitary MTP!) as reported on #41 , the phone which had only collected 30 MB had the app running, but wasn't actually running. Did a Force Stop and ran again. Now it is collecting data.
I think i will have to manually every few hours check the data eaten up by it.

One phone has consumed about 700 MB. Probably would need to copy its older data to my PC. Phone has about 350 MB free space left, about 5 days worth more data.

**Comment by nipunbatra** [comment]
Now if it try to open Funf, it will just keep on crashing! Dont know what to do.

**Comment by nipunbatra** [comment]
Restarted the phone and now data collection is again working fine.

**Comment by nipunbatra** [comment]
Another phone wasn't collecting data. In its case Data Collection button was **OFF**. 
Surely manually checking this is not the best method.
